tbx/core/blocks.py

```diff
@@ -82,37 +70,57 @@ class ImageFormatChoiceBlock(FieldBlock):
     pass
 
 
-class ImageBlock(StructBlock):
+class BaseImageStructValue(blocks.StructValue):
```

Comment:
Suggestion: rename to `AltTextStructValue` - or something that makes clear that this class relates to returning the alt text.